sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADOAuth2UserService.java

```diff
@@ -96,6 +100,9 @@ public OidcUser loadUser(OidcUserRequest userRequest) throws OAuth2Authenticatio
         if (authorities.isEmpty()) {
             authorities = DEFAULT_AUTHORITY_SET;
         }
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("User's authorities: {}.", authorities);
```

Comment:
Mark.

Reply:
`User's authorities extracted by id token and access token: {}.`

Reply:
OK